disnake/member.py

```diff
@@ -718,6 +738,26 @@ def flags(self) -> MemberFlags:
         """
         return MemberFlags._from_value(self._flags)
 
+    @property
+    def avatar_decoration(self) -> Optional[Asset]:
```

Comment:
Looks like this prevents people from accessing the underlying user's avatar decoration, if they have different global and guild-specific decorations set. It would need to be implemented like avatars, with `guild_avatar_decoration` and `display_avatar_decoration`(?).

Reply:
Nice catch 80a8f0da50581d17502d91a1d899e11c4c43a5f9